### DISCUSSION TITLE
The sunsetting of Quasar v1 has been postponed! 🙂

### DISCUSSION THREAD

**Discussion by smolinari** [quasarframework/quasar#13089] [Announcements]
Hey everyone...

The sunsetting of Quasar v1  has been postponed! 🙂

Quasar version 1's end-of-life has been postponed until December 31st of this year. We realize there are a good number users still working actively on larger apps, and we want to offer them more time to migrate to version 2. So, support for v1 will continue until, but also stop at the end of this year.

**Comment by susansiow** [comment]
Thank you for the super kind consideration, as it is not easy to make the transformation from a medium-sized (and above) application in a short period of time. Many thanks!
